Support_Page.php

```diff
@@ -17,8 +17,8 @@ static public function admin_print_scripts_w3tc_support() {
 		$w3tc_support_field_name = '';
 		$w3tc_support_field_value = '';
 
-		if ( isset( $_GET['service_item'] ) ) {
-			$pos = (int) $_GET['service_item'];
+		if ( isset( Util_Request::get_integer( 'service_item' ) ) ) {
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.